zb-db/src/main/java/io/camunda/zeebe/db/impl/DbForeignKey.java

```diff
@@ -23,9 +24,18 @@
  * @param columnFamily
  * @param <K>
  */
-public record DbForeignKey<K extends DbKey>(K inner, Enum<?> columnFamily)
+public record DbForeignKey<K extends DbKey>(
+    K inner, Enum<?> columnFamily, MatchType match, Predicate<K> skip)
```

Comment:
:question: I don't understand the use of the skip feature. I read it in the description, but I cannot recall any place where this would be used. Please give an example of where this would be used.

Reply:
For example here: https://github.com/camunda/zeebe/issues/8930#issuecomment-1074111759 where we use `-1` to represent something that does not exist.